packages/wdio-browserstack-service/tests/service.test.js

```diff
@@ -236,8 +313,7 @@ describe('_getBody', () => {
     it('should return "error" if failures', () => {
         service.failures = 1
         service.failReason = 'error message'
-        service.fullTitle = 'foo - bar',
-
+        ;(service.fullTitle = 'foo - bar'),
```

Comment:
could you tell me why you add these symbols `;(    ),` here? :)

Reply:
I didn’t notice they were there, could be my VS Code settings? I’ll fix it.